src/snapshots/001.txt

```diff
@@ -22,17 +23,17 @@ Newline(1:1-1:1),
 StringStart(8:5-8:6),StringPart(8:6-8:9),StringEnd(8:9-8:10),EndOfFile(8:10-8:10),
 ~~~PARSE
 (file
-    (module (1:1-4:7) (exposed_item (lower_ident "foo")))
-    (decl (6:1-8:10)
-        (ident (6:1-6:4) "foo")
-        (string (8:5-8:10) (string_part (8:6-8:9) "one"))))
+	(module (1:1-4:7) (exposed_item (lower_ident "foo")))
```

Comment:
I wonder if we can change github to render tabs with less space. They apparently do 8 character wide tabs by default.

Reply:
It's a personal setting, under Appearance

Reply:
I've set mine to 4